src/server/index.js

```diff
@@ -18,8 +18,8 @@ const server = http.createServer(app);
 require('./io').set(server);
 require('./routes/sockets');
 
-app.use(bodyParser.urlencoded({ extended: true}));
-app.use(bodyParser.json());
+app.use(bodyParser.json({limit: '50mb'}));
```

Comment:
How large of a payload is 10 rendered images?

Reply:
@d2fong  I was running into issues where very large pathways were causing payload size issues, so I increased it to 50mb just to avoid any possible errors during testing.

I have now changed it to a more realistic 5mb.